src/PrestaShopBundle/Entity/AdminFilter.php

```diff
@@ -83,7 +83,7 @@ class AdminFilter
     /**
      * @var string
      *
-     * @ORM\Column(name="filter_id", type="string", length=255)
+     * @ORM\Column(name="filter_id", type="string", length=191)
```

Comment:
Because of utf8mb4.
> InnoDB has a maximum index length of 767 bytes for tables that use COMPACT or REDUNDANT row format, so for utf8mb3 or utf8mb4 columns, you can index a maximum of 255 or 191 characters, respectively. If you currently have utf8mb3 columns with indexes longer than 191 characters, you must index a smaller number of characters.

Reply:
or specify explicitly index length
`CREATE INDEX idx_filter_id ON admin_filter (filter_id(191));`
for most cases if data fits into index lookup it will be fast. But if in most cases data exceeds 191 indexed chars it will have to do more work and filter more rows after applying the index

Reply:
This is also why it's not needed to have a `varchar(255)` with an `index(191)`, otherwise the index shouldn't be on this column :thinking: